docs/devices/HSE2919E.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Datek HSE2919E control via MQTT"
+description: "Integrate your Datek HSE2919E via Zigbee2MQTT with whatever smart home
+ infrastructure you are using without the vendors bridge or gateway."
+---
+
+*To contribute to this page, edit the following
+[file](https://github.com/Koenkk/zigbee2mqtt.io/blob/master/docs/devices/HSE2919E.md)*
+
+# Datek HSE2919E
+
+| Model | HSE2919E  |
+| Vendor  | Datek  |
+| Description | Water leak sensor |
+| Exposes | battery, battery_low, temperature, water_leak, tamper, linkquality |
+| Picture | ![Datek HSE2919E](../images/devices/HSE2919E.jpg) |
+
+## Notes
+
+This device support `iasZone` and `iasWd`
```

Comment:
I don't think this should be mentioned here, most users probably don't know what this means

Reply:
I can remove it if you like, but for those who know what it is and perhaps searching for devices supporting it (like I did), it is handy information. I've been contacted by other users about this after updating the devices to support this feature.

Your call Koen 👍

Reply:
I'm fine with it, but only if an explanation is added what it actually means (for the user)

Reply:
Ok, removed the "**This device support `iasZone` and `iasWd**`" as there is no easy way of explaining iasZone and iasWd.